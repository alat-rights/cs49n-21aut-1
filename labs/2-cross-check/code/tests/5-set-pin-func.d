5-set-pin-func.o: 5-set-pin-func.c ../rpi.h
