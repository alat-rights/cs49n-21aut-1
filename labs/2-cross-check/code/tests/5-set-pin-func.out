TRACE: calling pi code
TRACE: GET32(0x20200000) = 0x327b23c6
TRACE: PUT32(0x20200000) = 0x327b23c0
TRACE: GET32(0x20200000) = 0x327b23c0
TRACE: PUT32(0x20200000) = 0x327b23c8
TRACE: GET32(0x20200000) = 0x327b23c8
TRACE: PUT32(0x20200000) = 0x327b2288
TRACE: GET32(0x20200000) = 0x327b2288
TRACE: PUT32(0x20200000) = 0x327b2688
TRACE: GET32(0x20200000) = 0x327b2688
TRACE: PUT32(0x20200000) = 0x327b4688
TRACE: GET32(0x20200000) = 0x327b4688
TRACE: PUT32(0x20200000) = 0x327ac688
TRACE: GET32(0x20200000) = 0x327ac688
TRACE: PUT32(0x20200000) = 0x327ac688
TRACE: pi exited cleanly: 7 calls to random
